src/scripts/main.js

```diff
@@ -1,3 +1,247 @@
 'use strict';
 
 // write code here
+function formatSalary(salary) {
+  let value = salary.replace(/\D/g, '');
+
+  if (value) {
+    value = parseInt(value).toLocaleString('en-US');
+  }
+
+  return value;
+}
+
+const pushNotification = (
+  title,
+  description,
+  type,
+  posTop = 10,
+  posRight = 10,
+) => {
+  const body = document.querySelector('body');
+  const notification = document.createElement('div');
+  const messageTitle = document.createElement('h2');
+  const messageDescription = document.createElement('p');
+
+  notification.setAttribute('class', `notification ${type}`);
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+  notification.setAttribute('data-qa', 'notification');
+  notification.classList.add(type);
+
+  messageTitle.setAttribute('class', 'title');
+  messageTitle.innerText = title;
+  messageDescription.innerText = description;
+
+  notification.appendChild(messageTitle);
+  notification.appendChild(messageDescription);
+  body.appendChild(notification);
+
+  setTimeout(() => {
+    notification.style.display = 'none';
+  }, '2000');
```

Comment:
The timeout value for `setTimeout` should be a number, not a string. Change `'2000'` to `2000`.